app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -27,125 +24,79 @@ const PageResults: React.FunctionComponent<RouteComponentProps> = () => (
         return "edit/overview";
       }
 
-      if (analysis?.show_analysis === false) {
+      if (!analysisAvailable(analysis)) {
         return "design";
       }
     }}
   >
-    {({ experiment, analysis }) => (
-      <>
-        <MonitoringLink {...experiment} />
-        {analysis?.show_analysis ? (
-          <AnalysisAvailable {...{ experiment, analysis }} />
-        ) : (
-          <AnalysisUnavailable {...{ experiment, error: analysis == null }} />
-        )}
-      </>
-    )}
-  </AppLayoutWithExperiment>
-);
-
-const AnalysisAvailable = ({
-  experiment,
-  analysis,
-}: {
-  experiment: getExperiment_experimentBySlug;
-  analysis: AnalysisData | undefined;
-}) => {
-  const slugUnderscored = experiment.slug.replace(/-/g, "_");
-  return (
-    <>
-      <h3 className="h5 mb-3 mt-4">Overview</h3>
-      <p className="mb-4">
-        Detailed analysis{" "}
-        <LinkExternal
-          href={`https://protosaur.dev/partybal/${slugUnderscored}.html`}
-          data-testid="link-external-results"
-        >
-          can be found here
-        </LinkExternal>
-        .
-      </p>
-      <h3 className="h6">Hypothesis</h3>
-      <p>{experiment.hypothesis}</p>
-      <TableHighlights
-        primaryProbeSets={experiment.primaryProbeSets!}
-        results={analysis?.overall!}
-      />
-      <TableHighlightsOverview
-        {...{ experiment }}
-        results={analysis?.overall!}
-      />
+    {({ experiment, analysis }) => {
+      // For testing - users will be redirected if the analysis is unavailable
+      // before reaching this return, but tests reach this return and
+      // analysis.overall is expected to be an object
+      if (!analysisAvailable(analysis)) return <></>;
```

Comment:
This could also be `analysis?.overall`, or stored in a var to only run the function once (it's super lightweight though) but finding a work around to remove these lines would be much nicer. 🙃